docs/internals/contributing/writing-documentation.txt

```diff
@@ -256,6 +256,23 @@ If a function, attribute, etc. is added, it's also okay to use a
 We can simply remove the ``.. versionadded:: A.B`` annotation without any
 indentation changes when the time comes.
 
+Minimizing images
+-----------------
+
+Optimize image compression where possible. For PNG files, try OptiPNG and
+AdvanceCOMP's ``advpng``:
+
+.. code-block:: console
+
+   $ cd docs/
+   $ optipng -o7 -zm1-9 -i0 -strip all `find . -type f -name "*.png"`
+   $ advpng -z4 -i 1000 `find . -type f -name "*.png"`
+
+.. note::
+
+   This is based on OptiPNG version 0.7.5. Older versions may complain about
```

Comment:
I just checked their history http://optipng.sourceforge.net/history.txt and it appears the "--strip all" function was added in 0.7.0...